build-system/tasks/check-owners.js

```diff
@@ -22,28 +22,66 @@
 
 'use strict';
 
+const argv = require('minimist')(process.argv.slice(2));
+const BBPromise = require('bluebird');
```

Comment:
We’re trying to replace `BBPromise` with `utils.promisify` across the code base. Can you try using that instead?

Reply:
I see bluebird used everywhere throughout the codebase, while utils.promisify doesn't show up in a search. I feel like, since the migration may be intended but is not yet underway, it makes sense to keep things consistent and then it can be updated in the eventual find+replace PR. WDYT?

Reply:
This is being done in #25822.